src/pages/App.css

```diff
@@ -0,0 +1,50 @@
+.body{
```

Comment:
the css spec are only for mortgage application, rename it the mortgage.css and imported by mortgage/index only

Reply:
remove the change here